packages/pglite/src/parse.ts

```diff
@@ -68,13 +79,14 @@ export function parseResults(
   return resultSets;
 }
 
-function affectedRows(msg: CommandCompleteMessage): number {
+function retrieveRowCount(msg: CommandCompleteMessage): number {
   const parts = msg.text.split(" ");
   switch (parts[0]) {
     case "INSERT":
+      return parseInt(parts[2], 10);
     case "UPDATE":
     case "DELETE":
-      return parseInt(parts[1]);
+      return parseInt(parts[1], 10);
```

Comment:
changed to `retrieveRowCount`, because I realized that `INSERT` statements send back a message that looks like `INSERT 0 1`, where the _second_ number is actually the count of rows inserted. Per our previous conversation, if we want an `affectedRows` count of `2`, then we would need to consider that number.